frappe/templates/emails/footer_template.html

```diff
@@ -0,0 +1,21 @@
+<!-- Footer block -->
+<tr>
+	<td align="center" valign="top" class="footer">
+		<table width="100%" border="0" cellpadding="0" cellspacing="0">
+			<tbody>
+				<tr><td style="height: 20px;"></td></tr>
+				<tr>
+					<td valign="top" width="100%">
+						<h6>Sent with &#9829; from ERPNext</h6>
```

Comment:
we don't want erpnext hardcoded. this should be the company who is sending the emails. in frappe it should just be some standard text.